README.md

```diff
@@ -125,4 +125,26 @@ Run single Playwright tests
 
 ```
 pnpm exec playwright test (name of file)
-```
\ No newline at end of file
+```
+## Prettier
+
+Install dependencies if you have not done so already.
```

Comment:
Do not need to install. This should be higher up in the Doc already to install all dependencies.

Reply:
I edited the `README` to remove the part under the Prettier section regarding installing dependencies.